packages/plugins/io-center/src/query.svelte.ts

```diff
@@ -24,7 +24,7 @@ export function useQuery() {
 	$effect(() => storeLogicalConditioners(store.doc, store.selectedIED, store.editCount))
 	$effect(() => storeLogicalPhysicals(store.doc, store.selectedIED, store.editCount))
 	$effect(() => storeSelectedDataObjects(store.doc, store.selectedIED, store.editCount))
-	$effect(() => storeSelectedLogicalPhysicals(store.doc, store.selectedIED, store.editCount))
+	// $effect(() => storeSelectedLogicalPhysicals(store.doc, store.selectedIED, store.editCount))
```

Comment:
will be activated later when we implement the logic of showing only things that are relevant to the selected objects